app/models/postcode_local_restriction_search.rb

```diff
@@ -1,16 +1,16 @@
 class PostcodeLocalRestrictionSearch
   UK_POSTCODE_PATTERN = %r{
     \A
-    ([Gg][Ii][Rr] 0[Aa]{2}) # Unusual postcodes
+    (GIR\s?0AA) # Unusual postcodes
```

Comment:
Oh! So this isn't actually a postcode, it's for Girobank! https://www.streetcheck.co.uk/postcode/gir0aa
Could that be added to the commit message/comment?

Reply:
As it's not a geographical area, do we need to handle it? Mapit returns an error for this postcode regardless, so we're not really saving the user anything.